src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -83,6 +102,9 @@ public LocalCall(String functionName, Optional<List<?>> arg,
         arg.ifPresent(arg -> payload.put("arg", arg));
         kwarg.ifPresent(kwarg -> payload.put("kwarg", kwarg));
         metadata.ifPresent(m -> payload.put("metadata", m));
+        timeout.ifPresent(timeout -> payload.put("timeout", timeout));
```

Comment:
@lucidd - This PR fixes `local_batch` client: https://github.com/saltstack/salt/pull/40300